apps/meteor/client/components/GenericModal/GenericModal.tsx

```diff
@@ -98,9 +100,9 @@ const GenericModal = ({
 	useEffect(
 		() => () => {
 			if (!dismissedRef.current) return;
-			onClose?.();
+			onDismiss?.();
 		},
-		[onClose],
+		[onDismiss],
```

Comment:
If `onDismiss` is replacing `onClose`, shoudn't it receive `onClose` as default value?

Reply:
That's right! Thanks man